src/ProjectTemplates/Web.Spa.ProjectTemplates/content/React-CSharp/.template.config/template.json

```diff
@@ -34,6 +34,7 @@
             "Data/**",
             "Models/**",
             "ClientApp/src/components/api-authorization/**",
+            "ClientApp/src/setupTests.js",
```

Comment:
This `exclude`-based approach looks kind of risky, as in the future we might add more files and not realise they also have to be included in the `exclude` list. Two possibilities:

 * Do we already have tests that assert the template output *doesn't* include any extra unexpected files? If so we're probably fine already.
 * If not, maybe we should establish a naming convention like `internal.setupTests.js` and then have an exclude pattern like `**\internal.*` so someone who's just copying/pasting existing patterns would automatically have their new files exclued.

Not a massively big deal - just wanted to get your thoughts on this.

Reply:
Oh it looks like we do have baseline tests.